en/getting_started/faq.md

```diff
@@ -24,7 +24,7 @@ The architecture has stubs for serial, TCP, and UDP connections. However, only U
 
 ### Why is libCURL a dependency?
 
-libCURL will be required to download the camera definition file which is referenced in [CAMERA_INFORMATION](http://mavlink.org/messages/common#CAMERA_INFORMATION). It might also come in handy whenever any other REST resources are required.
+libCURL will be required to download the camera definition file which is referenced in [CAMERA_INFORMATION](http://mavlink.org/messages/common#CAMERA_INFORMATION). It might also come in handy whenever any other REST resources are required. 
```

Comment:
@julianoes  Where does DroneCore use CAMERA_INFORMATION?

Reply:
Hehe, that's still proprietary but I'll open source that camera plugin soon.

Reply:
Cool. Created https://github.com/dronecore/docs/issues/90 just to make sure this doesn't fall off the radar